base/record_replay.h

```diff
@@ -74,6 +74,10 @@ void BeginDisallowEvents();
 void BeginDisallowEventsWithLabel(const char* label);
 
 void EndDisallowEvents();
+
+// Whether we are in a code path that we have explicitely identified as
+// divergent. Those are generally code paths that either belong to the runtime
+// or Replay's intermittent command handling.
```

Comment:
Disallowed events are used for a lot more than command handling, this sentence should be removed.

Reply:
The first part is "the paths either belong to the runtime".
But sure, I'll remove it.